extension.json

```diff
@@ -37,6 +37,29 @@
         "type": "object",
         "properties": {}
       }
+    },
+    {
+      "displayName": "Auto Open Your App",
+      "name": "autoopenapp",
+      "libPath": "src/lib/actions/autoOpen.js",
```

Comment:
should this action be renamed to linkCreator as well?

Reply:
This is the action that will call .deepview() from the WebSDK so based on that, it has the correct name. The next phase will introduce a new action for sending SMSes which we will place in sms.js.